terraform/test/main.tf

```diff
@@ -106,10 +106,21 @@ module "marklogic" {
   environment     = "test"
   vpc             = module.networking.vpc
   private_subnets = module.networking.ml_private_subnets
-  instance_type   = "t3.xlarge"
+  instance_type   = "t3.large"
```

Comment:
I've only done an instance refresh for one of the ASGs, but it seemed to work fine.

Reply:
One thought from that: should we move the MarkLogic data EBS volumes into Terraform from CloudFormation? We really don't want to accidentally destroy them and Terraform gives us better controls for that.

Reply:
That's a good point. I'm surprised that the CF template doesn't have a deletion policy set.